pulsar-client-all/pom.xml

```diff
@@ -98,6 +98,7 @@
                 <includes>
                   <include>org.apache.pulsar:pulsar-client-original</include>
                   <include>org.apache.pulsar:pulsar-client-admin-original</include>
+                  <include>org.apache.pulsar:pulsar-client-api</include>
```

Comment:
I'd avoid this inclusion if possible. The purpose of extracting `pulsar-client-api` in a different module was to have an unshaded module so that it would have the sources and javadoc available in Maven repository (and displayed in the IDE).

Reply:
The reason for this inclusion is that the pulsar-client-api contains a package that is also contained in the pulsar-common module (transitive include from the other). So when pulsar-client-api is included in the shaded module, that will merge the split-package into the same jar file hence removing the split-package issue.

But, it would be even better if we had a solution without using shaded jar and making it work with the java-module-system.